C/cddlib/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cddlib"
+version = v"0.94j-6d980f22"
```

Comment:
Can this simply be
```suggestion
version = v"0.94"
```
?  `Pkg` doesn't really like when we use something in addition to the standard major.minor.patch scheme, this would be parsed as
```julia
julia> dump(v"0.94j-6d980f22")
VersionNumber
  major: UInt32 0x00000000
  minor: UInt32 0x0000005e
  patch: UInt32 0x00000000
  prerelease: Tuple{String}
    1: String "j-6d980f22"
  build: Tuple{} ()
```

Reply:
The j is the upstream versioning and there is an old `0.94` from 2005 which might be confusing for users seeing this. So I think we should either map it (more or less canonically) to `0.94.10` or use totally unrelated numbers like `0.1.0`, but I prefer to have the upstream version encoded there so I added the `.10`.
I also dropped the git hash.

Reply:
Yes, in general we prefer to use a version as close as possible to the upstream project, but we also have to deal with Pkg requirements.